src/main/java/reposense/model/ConfigCliArguments.java

```diff
@@ -17,14 +17,16 @@
     private Path authorConfigFilePath;
 
     public ConfigCliArguments(Path configFolderPath,
-            Path outputFilePath, Optional<Date> sinceDate, Optional<Date> untilDate, List<String> formats) {
+            Path outputFilePath, Optional<Date> sinceDate, Optional<Date> untilDate, List<String> formats,
+                              boolean isAutomaticallyLaunching) {
```

Comment:
Line break can be improved.
Move everything up a line then break after `sinceDate`

```
    public ConfigCliArguments(Path configFolderPath, Path outputFilePath, Optional<Date> sinceDate, 
            Optional<Date> untilDate, List<String> formats, boolean isAutomaticallyLaunching) {
```

Reply:
@myteo 

Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line. 
`From https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout`

You can copy the above code snip exactly or refer to [this line](https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/parser/AuthorConfigCsvParser.java#L78)

Reply:
Got it. Thanks